SushiRunner/Views/Home/MealDescription.cshtml

```diff
@@ -0,0 +1,30 @@
+@model SushiRunner.ViewModels.Home.MealDescriptionModel
+
+@{
+    ViewData["Title"] = "Sushi Runner";
+     Layout = "_HomeLayout";
+}
+
+<div>
+    <a asp-action="Index"> Back</a>
+</div>
+<div class="item-img">
+    <img src="@Model.ImagePath" alt="@Model.Name">
+</div>
+<div style="margin: auto; width: 80%;">
```

Comment:
Inline styles are bad.

Reply:
Also I told you to wrap the whole page in this div, you forgot about the image above and the price below